src/components/report/ReportChapterTableResult.svelte

```diff
@@ -48,7 +48,7 @@
       <ul>
         {#each criteria.components as component}
           {#if component.adherence.level}
-            <li>{@html catalogComponentLabel(catalogName, component.name, "html")}<p>{levelLabel(catalogName, component.adherence.level)}</p></li>
+            <li><!-- {@html catalogComponentLabel(catalogName, component.name, "html")} --><p>{levelLabel(catalogName, component.adherence.level)}</p></li>
```

Comment:
So these are just commented out because??

Reply:
See comment https://github.com/CivicActions/openacr-editor/pull/1#discussion_r1104547904. I have a solution for it in the other merge request to allow it to be kept.

Reply:
Solution in place and also in the CLI